modules/business-text/business-text.graphql.js

```diff
@@ -0,0 +1,20 @@
+const businessTextType = `
+    type BusinessText {
+    _id: ID!
+    code: String
```

Comment:
I don't think that it should be nullable

Reply:
done